packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -41,6 +43,8 @@ class ResizeHandle extends (Widget as any) {
       showResizableIcon: true,
       showCollapsePrev: true,
       showCollapseNext: true,
+      collapsePrevClick: null,
```

Comment:
May be it's better name this options as 'onCollapsePrev', 'onCollapseNext'. What do you think?